sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -371,6 +371,14 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val DEFAULT_PARALLELISM = buildConf("spark.sql.default.parallelism")
```

Comment:
`spark.sql.default.parallelism` -> `spark.sql.sessionLocalDefaultParallelism`?

Reply:
Em.. is it better to keep similar with `spark.default.parallelism`? so we can set this config easy. `sessionLocalDefaultParallelism ` seems complex.